client/packages/common/src/intl/locales/en/inventory.json

```diff
@@ -7,7 +7,11 @@
   "error.no-stocktakes": "There are no Stocktakes to display.",
   "error.no-stocktake-items": "No items have been added to this stocktake.",
   "error.stocktake-not-found": "Stocktake not found",
+  "error.not-editable": "Stocktake is not editable.",
+  "error.is-locked": "Stocktake is locked.",
+  "error.snapshot-total-mismatch": "Snapshot quantity does not match total quantity.",
   "error.provide-reason": "A reason must be provided for any rows which have a difference in the counted number of packs.",
+  "error.provide-valid-reason": "Adjustment reason does not match adjustment direction",
   "error.reduced-below-zero": "Stock line exist in new outbound shipments. The quantity cannot be reduced below zero.",
   "error.stocktake-has-stock-reduced-below-zero": "Stock take cannot be finalised because there are stock lines which exist in new outbound shipments.",
```

Comment:
```suggestion
  "error.stocktake-has-stock-reduced-below-zero": "Stock take cannot be finalised because some of the stock has been used in new outbound shipments.",
```

Reply:
🙏 much better